hypershift-operator/controllers/nodepool/nodepool_controller.go

```diff
@@ -1307,11 +1318,10 @@ func validateAutoscaling(nodePool *hyperv1.NodePool) error {
 	return nil
 }
 
-func defaultNodePoolAMI(region string, releaseImage *releaseinfo.ReleaseImage) (string, error) {
-	// TODO: The architecture should be specified from the API
-	arch, foundArch := releaseImage.StreamMetadata.Architectures["x86_64"]
+func defaultNodePoolAMI(region string, nodepoolArch string, releaseImage *releaseinfo.ReleaseImage) (string, error) {
```

Comment:
let's name it just arch, nodepool is the context itself so is redundant. Otherwise every other variable would need to be prefixed as well.

Reply:
Will do. I missed that one when cleaning up the name.

Reply:
I named it something else. `arch` is already used in that function.